### PR TITLE
add voucher code in success message

### PR DESCRIPTION
**Pull request by jlwt90**
Hello, this is the PR for #390 .
:D

### CODE REVIEW

File: src/pretix/control/views/vouchers.py [hunk 1]
```diff
@@ -215,7 +215,7 @@ def get_form_kwargs(self):
     @transaction.atomic
     def form_valid(self, form):
         form.instance.event = self.request.event
-        messages.success(self.request, _('The new voucher has been created.'))
+        messages.success(self.request, _('The new voucher has been created.') + ' (%s)' % form.instance.code)
```
Comment (raphaelm) [inline src/pretix/control/views/vouchers.py]: I'd suggest a more translatable form here, as the style with the parantheses might not be appropriate in every language.

```
 messages.success(self.request, _('The new voucher has been created: {}').format(form.instance.code))
```

File: src/tests/control/test_vouchers.py [hunk 2]
```diff
@@ -133,6 +133,18 @@ def test_bulk_rng(self):
...
+        count_before = self.event.vouchers.count()
+        doc = self.get_doc('/control/event/%s/%s/vouchers/add' % (self.orga.slug, self.event.slug))
+        form_data = extract_form_fields(doc.select('.container-fluid form')[0])
+        form_data.update({
+            'itemvar': '%d' % self.ticket.pk
+        })
+        doc = self.post_doc('/control/event/%s/%s/vouchers/add' % (self.orga.slug, self.event.slug), form_data)
+        v = Voucher.objects.latest('pk')
+        assert v.code in doc.select(".alert-success")[0].text
+        assert count_before + 1 == self.event.vouchers.count()
```
Comment (raphaelm) [inline src/tests/control/test_vouchers.py]: Nice!

File: src/pretix/control/views/vouchers.py (hunk 1 above)
Comment (jlwt90) [inline src/pretix/control/views/vouchers.py]: nice idea! Just a quick question, I think I need to update the translation as well?

./pretix/locale/de/LC_MESSAGES/django.po
./pretix/locale/de_Informal/LC_MESSAGES/django.po
```
#: pretix/control/views/vouchers.py:218
msgid "The new voucher has been created."
msgstr "Der neue Gutschein wurde erstellt."
```

Comment (jlwt90) [inline src/pretix/control/views/vouchers.py]: updated the line above.
```
messages.success(self.request, _('The new voucher has been created: {code}').format(code=form.instance.code))
```
I also updated the translation files at the same time.

**Comment by raphaelm** [comment]
Nice, thank you very much!

**Pull request merged**